README.md

```diff
@@ -134,7 +134,7 @@ Coming soon
 
```

Comment:
- [ ] Kindly improve the README file by completing the following: `Install` and `Usage` Section. In the Install section, we can install postgresql, without it we can not run the app as seen in the screenshot . Also, how to use the app should be stated in the README file under the usage section, since we have run in the terminal we should be able to add it to the file the command used for running the app
![Screenshot from 2023-08-16 16-54-32](https://github.com/francksefu/vet-R/assets/46388113/014f0ff5-e0d5-4cfb-aa0b-8746cf026c1b)